RSSReader-Maruyama/DataManager/UserDataManager.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  LoginChecker.swift
+//  RSSReader-Maruyama
+//
+//  Created by 丸山翔 on 2020/09/03.
+//  Copyright © 2020 丸山翔. All rights reserved.
+//
+
+import UIKit
+
+class UserDataManager: NSObject {
+    
+    private let UD_USER_DATA_KEY = "userdata"
+    lazy var userDefault = UserDefaults.standard
+
+    
+    func isLoggedIn() -> Bool {
+        let user = userDefault.object(forKey: UD_USER_DATA_KEY ) as? Data
```

Comment:
```suggestion
        let user = userDefaults.data(forKey: USER_DATA_KEY)
```
dataにキャストする処理は不要です。
UserDefaultsにData型でデータを取得できるメソッドが用意されているのでそちらを使用してください。

Reply:
修正しました。